settings.gradle.kts

```diff
@@ -118,6 +118,14 @@ plugins {
 
 dependencyResolutionManagement {
     repositories {
+        mavenLocal {
+            name = "localOrt"
+
+            content {
+                includeGroupByRegex("org\\.ossreviewtoolkit(\\..*)?")
```

Comment:
I guess you could use `includeGroupAndSubgroups()` (or just `includeGroup()`) instead.